### ISSUE TITLE
Windows on a secondary screen

### ISSUE THREAD

**Issue by bernie79160** [mainIine/foe-helfer-extension#2673]
Are you going to develop the window display function on a secondary screen?

**Comment by SkateScout** [comment]
Hi, this is technically not possible with browser functions.

**Comment by bernie79160** [comment]
I have difficulty understanding, but these are really dialog boxes?
what's stopping them from being put on a second screen?

I point out that I'm not a developer so I may be asking the impossible, but I do this with a lot of other games, that's why I asked this question.

**Comment by bernie79160** [comment]
This is what I found on the Internet
Windows and Pop-ups: You can use JavaScript to open new windows or pop-ups in Microsoft Edge. However, how these windows are displayed will depend on the underlying operating system (like Windows). You can control the size and position of these windows using the appropriate parameters in the window.open() function.

Multi-Monitor: Windows, the most commonly used operating system with Microsoft Edge, supports managing multiple displays. You can extend or duplicate your desktop across multiple displays using operating system settings.

User Interface Manipulation: You can use JavaScript to move and resize windows, but this will also depend on browser permissions and settings.

System Features: You can use specific operating system features, such as external display management, to determine where windows will be displayed.

In summary, although Microsoft Edge can be used to create windows and dialog boxes, how they are displayed on a second screen will mainly depend on how the operating system is handled. You can use browser and operating system features to determine where and how these windows appear on multiple screens.

**Comment by Beelzebob6666** [comment]
As far as  heard, it technically might be possible to do it, but it is quite a challenge as some kind of communication has to be established between the main window and the secondary window as that is not a readily available browser feature.

As far as I heard it has been tried in the past already but was given up at some point - not sure if it was an issue of time constraint or feasability.

**Comment by outoftheline** [comment]
We will not continue investigating this for now
